homepage.html

```diff
@@ -12,11 +12,46 @@
         h1
         {
             position: relative;
-            top: 50%;
+            top: 30%;
             font-size: 100px;
         }
+        p
+        {
+            position: relative;
+            top: 50%;
+            font-size: 30px;
+        }
+        div
+        {
+            position: relative;
+            top: 20%;
+            font-size: 50px;
+        }
+        button
+        {
+            font-size: 30px;
+        }
     </style>
     <body>
-        <h1>WELCOME</h1>
+        <h1>WELCOME!</h1>
+        <p><a href="http://127.0.0.1:5500/loginlink.html"><button>LOGIN</button></a></p>
+        <div>
+            <script type="text/javascript">
```

Comment:
Put javascript code in a separate folder and refer the script from there. Study about the methods to invoke javascript code inside a html file.